frontend/src/features/comparisons/Comparison.tsx

```diff
@@ -215,40 +215,18 @@ const Comparison = ({
   };
 
   return (
-    <Grid
-      container
-      sx={{
-        maxWidth: '880px',
-        gap: '8px',
-      }}
-    >
-      <Grid
-        item
-        xs
-        component={Card}
-        display="flex"
-        flexDirection="column"
-        alignSelf="stretch"
-      >
+    <Grid container maxWidth="880px" gap={1}>
+      <Grid item xs display="flex" flexDirection="column" alignSelf="stretch">
```

Comment:
(minor) Removing the wrapping `Card` and the title causes the "full" variant of the selector buttons to be misaligned with the other selector:

![image](https://github.com/tournesol-app/tournesol/assets/4726554/3febb797-9fb7-4c6f-8cc0-e89d7141eb1c)

(Visible after erasing the content of one of the video selectors in the video_id text field)

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1929/commits/29a5204781cb5e5ef3fbf73327bf99d3ea8c24df